src/transformers/modeling_outputs.py

```diff
@@ -55,10 +74,7 @@ class BaseModelOutputWithPooling(ModelOutput):
         last_hidden_state (`torch.FloatTensor` of shape `(batch_size, sequence_length, hidden_size)`):
             Sequence of hidden-states at the output of the last layer of the model.
         pooler_output (`torch.FloatTensor` of shape `(batch_size, hidden_size)`):
-            Last layer hidden-state of the first token of the sequence (classification token) after further processing
-            through the layers used for the auxiliary pretraining task. E.g. for BERT-family of models, this returns
-            the classification token after processing through a linear layer and a tanh activation function. The linear
-            layer weights are trained from the next sentence prediction (classification) objective during pretraining.
+            Last layer hidden-state after a pooling operation on the spatial dimensions.
```

Comment:
Hmm that might be a bit too limited.

cc'ing @patrickvonplaten here - this change would be made to make `BaseModelOutputWithPooling` also useful for vision models, for which this docstring isn't applicable.

Reply:
Ah good catch, I wanted that change on the `BaseModelOutputWithNoAttentionAndWithPooling`, not this class, which should be left untouched.

Reply:
Thanks for the comments! The doc is also updated in `BaseModelOutputWithNoAttentionAndWithPooling`. We can also be more general and just say ` Last layer hidden-state after a pooling operation.`

Reply:
Yes but you can actually revert this change here :)

Reply:
oh okay, I see! I got confused by my own names! Thanks